src/cli/commands/init.js

```diff
@@ -17,6 +17,12 @@ module.exports = Command.extend({
 
     function bootstrap () {
       var config = {
+        provider:{
+          host: 'ipfs.infura.io',
```

Comment:
Does Infura have R+W nodes always available?

Reply:
@diasdavid yes, infura has good uptime http://status.infura.io/2286441

Reply:
this is default config, for users can install and use, they allways can change provider in ipscend.json